samclip/0.4.0/Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG SAMCLIP_VER="0.4.0"
+
+FROM ubuntu:jammy as app
+
+ARG SAMCLIP_VER
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="samclip"
+LABEL software.version="${SAMCLIP_VER}"
+LABEL description="Samclip: filter SAM file for soft and hard clipped alignments"
+LABEL website="https://github.com/tseemann/samclip"
+LABEL license="https://github.com/tseemann/samclip/blob/master/LICENSE"
+LABEL maintainer="Dhatri Badri"
+LABEL maintainer.email="dhatrib@umich.edu"
+
+# install dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    wget \
+    software-properties-common \
+    perl && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# install samclip
+RUN wget https://raw.githubusercontent.com/tseemann/samclip/master/samclip && \
+    chmod +x samclip && \
+    mkdir /data
```

Comment:
This looks like a good start, but right now there's no version listed. Each version listed on github actually has a tar.gz or zip file that captures the files in that repository at that time. For samclip they are found here : https://github.com/tseemann/samclip/releases

Could you change this line to something like
```
RUN wget https://github.com/tseemann/samclip/archive/refs/tags/v${SAMCLIP_VER}.tar.gz && \
-xvf v${SAMCLIP_VER}.tar.gz && \
rm v${SAMCLIP_VER}.tar.gz && \
chmod +x /samclip-${SAMCLIP_VER}/samclip && \
mkdir /data
```

This would corresponding to your PATH adjustment on line 30.

Reply:
I have made the change you requested!